File: DSA/Ayush_Mishra_590014844/Day_32_Question_2.c
#include <stdio.h>

int main() {
    int n;
    printf("Enter the number of elements in the array: ");
    scanf("%d", &n);

    int arr[n], result = 0;
    printf("Enter %d elements:\n", n);
    for (int i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
        result ^= arr[i];
    }

    printf("The non-repeating element is: %d\n", result);
    return 0;
}

File: DSA/Nivedan_590014904/Day_32_Question_2.py
arr = []
n = int(input("Enter the number of elements in array:- "))
for i in range(n):
    val = int(input(f"Enter the {i} element: "))
    arr.append(val)
for i in range(len(arr)):
    for j in range(len(arr)):
        if arr[i]^arr[j]==0 and i!=j:
            break
    else:
        print(arr[i])
